librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -3226,9 +3234,12 @@ void QC_ApplicationWindow::widgetOptionsDialog()
         settings.setValue("AllowStatusbarHeight", allow_statusbar_height);
         if (allow_statusbar_height)
         {
-            int statusbar_height = dlg.statusbar_height_spinbox->value();
+            statusbar_height = dlg.statusbar_height_spinbox->value();
             settings.setValue("StatusbarHeight", statusbar_height);
-            statusBar()->setMinimumHeight(statusbar_height);
+            statusBar()->setFixedHeight(statusbar_height);
+
+            mouseWidget->findChild<QLabel*>("lMousePixmap")->setPixmap(
```

Comment:
This is *BAD* (commit v3). It basically rescales the pixmap image, but relatively. So after scaling down, and then rescaling up, it rasterizes the image. 

Fixed in commit v4.